tests/test_similar.py

```diff
@@ -91,6 +92,7 @@ def test_duplicate_code_raw_strings_disable_file(self) -> None:
                 "--enable=duplicate-code",
                 "--ignore-imports=no",
                 "--ignore-signatures=no",
+                "--min-similarity-lines=4",
```

Comment:
At this point we could use a pylintrc for similar tests instead of duplicating the option in all calls ?

Reply:
Yeah I thought about doing something like that, but then I thought: it's actually better to be explicit about this. Kind of strange how us changing our `pylintrc` is breaking tests. This makes it much clearer what we are actually testing.

Reply:
> Kind of strange how us changing our pylintrc is breaking tests.

I agree that's why I did #6459, it should not happen again (I'm surprised it does as I tested this by raising an exception in init-hook). What I mean is create a specific pylintrc for the similar tests. But it's a little less explicit than calling all the options in the tests itself, you do have to go check the content of the pylintrc.

Reply:
Yeah, I think this is fine for now. I promise to revisit when our issue count is less than 10 😄

Reply:
Haha, good one 😄 We're getting closer and closer to under 700 though !